java/pom.xml

```diff
@@ -37,6 +37,18 @@
     <tag>HEAD</tag>
   </scm>
   <dependencies>
+    <dependency>
```

Comment:
Question: will those classes on the two dependencies will be added to the final release artifact?